docs/source/python_api/mlflow.metrics.rst

```diff
@@ -31,16 +35,41 @@ The following code demonstrates how to use :py:func:`mlflow.evaluate()` with a i
         logged_model.model_uri,
         eval_df,
         model_type="text",
-        custom_metrics=[correctness_metric],
+        extra_metrics=[correctness_metric],
```

Comment:
I think we need `targets` here for ground_truth right?

Also can we have a sample eval_df signature so folks know an example format?